utoipa-rapidoc/Cargo.toml

```diff
@@ -23,5 +23,8 @@ actix-web = { version = "4", optional = true, default-features = false }
 rocket = { version = "0.5", features = ["json"], optional = true }
 axum = { version = "0.7", default-features = false, features = ["json"], optional = true }
 
+[dev-dependencies]
+utoipa-rapidoc = { path = ".", features = ["actix-web", "axum", "rocket"] }
```

Comment:
there is another way to solve this

https://github.com/async-rs/async-std/blob/main/src/lib.rs#L140-L141

```
#![cfg_attr(feature = "attributes", doc = "```")]
#![cfg_attr(not(feature = "attributes"), doc = "```ignore")]
```

That will take a bit longer

Reply:
So this ignores certain parts of the docs for it won't run the doc tests? Interesting concept probably could be leveraged on some cases through. :thinking:

Reply:
Right - that approach means the doctests are optionally activated by feature flags as well, so all the features dont need to all added to dev-dependencies .

I havent done this before - I just saw the trick last weekend and it looks like it will help here.

Reply:
Neat, it probably could.